sympy/solvers/solveset.py

```diff
@@ -889,33 +696,49 @@ def solveset(f, symbol=None, domain=S.Complexes):
     Examples
     ========
 
-    >>> from sympy import exp, Symbol, Eq, pprint, S, solveset
-    >>> from sympy.abc import x
+    >>> from sympy import exp, sin, Symbol, pprint, S
+    >>> from sympy.solvers.solveset import solveset, solveset_real
+
+    * The default domain is complex. Not specifying a domain will lead
+      to the solving of the equation in the complex domain (and this
+      is not affected by the assumptions on the symbol):
 
-    * The default domain is complex. Not specifying a domain will lead to the
-      solving of the equation in the complex domain.
+    >>> x = Symbol('x')
+    >>> pprint(solveset(exp(x) - 1, x), use_unicode=False)
+    {2*n*I*pi | n in Integers()}
 
+    >>> x = Symbol('x', real=True)
     >>> pprint(solveset(exp(x) - 1, x), use_unicode=False)
     {2*n*I*pi | n in Integers()}
 
-    * If you want to solve equation in real domain by the `solveset`
-      interface, then specify that the domain is real. Alternatively use
-      `solveset\_real`.
+    * If you want to use `solveset` to solve the equation in the
+      real domain, provide a real domain. (Using `solveset\_real`
+      does this automatically.)
 
+    >>> R = S.Reals
```

Comment:
This is a useful alias. We should create a PR so that someone will be able to import it by `from sympy import I` like we can do for `I`, `E` and `pi`.